modules/justpremiumBidAdapter.js

```diff
@@ -2,12 +2,14 @@ import { registerBidder } from '../src/adapters/bidderFactory.js'
 import { deepAccess } from '../src/utils.js';
 
 const BIDDER_CODE = 'justpremium'
+const GVLID = 62;
```

Comment:
This block seems to have no semicolons, probably introducing a bunch of lgtm warnings

Reply:
Ah! I missed one...
```suggestion
const GVLID = 62
```